Description: Try to dlopen curl-gnutls
 Since we link with gnutls, try to dlopen curl-gnutls (in addition to the
 OpenSSL variant).
Author: Andreas Metzler <ametzler@debian.org>
Origin: other
Forwarded: no
Last-Update: 2015-09-13

--- a/src/lib/ecore_con/ecore_con_url.c
+++ b/src/lib/ecore_con/ecore_con_url.c
@@ -320,6 +320,9 @@
    LOAD("libcurl.so.5"); // if fail try 2
    LOAD("libcurl.so.4"); // if fail try 2
 #else
+# ifdef HAVE_GNUTLS
+   LOAD("libcurl-gnutls.so.4"); // try this one first
+# endif /* ifdef HAVE_GNUTLS */
    LOAD("libcurl.so.5"); // try only
    LOAD("libcurl.so.4"); // try only
 #endif
